cli/src/commands/schema/mod.rs

```diff
@@ -4,10 +4,46 @@ pub mod store;
 
 use structopt::StructOpt;
 
+use crate::errors::{ErrorDetails, Fallible};
 pub use check::Check;
 pub use get::Get;
+use regex::Regex;
 pub use store::Store;
 
+enum SchemaRef {
+    SchemaVariantRef { graph_id: String, variant: String },
+    SchemaHashRef { graph_id: String, hash: String },
+}
+
+fn parse_schema_ref(src: &str) -> Fallible<SchemaRef> {
```

Comment:
This is ideal code for unit testing instead of capturing these in integration tests.

Reply:
this is really the only "request changes" comment.